src/reanimated2/mappers.ts

```diff
@@ -115,7 +115,10 @@ export function createMapperRegistry() {
       }
     } else if (inputs.addListener) {
       resultArray.push(inputs);
-    } else if (typeof inputs === 'object') {
+    } else if (inputs.__proto__ === Object.prototype) {
```

Comment:
The `__proto__` property seems to be deprecated, maybe could we use [`Object.getPrototypeOf`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/getPrototypeOf)?

Reply:
sure, that's a good suggestion